### PR TITLE
feat(dgw): add analytics to gateway standalone

### PR DESCRIPTION
**Pull request by irvingoujAtDevolution**
Enable analytics on Gateway Standalone with OpenSearch.

Place to pay attention to:
There is really no elegant solution to use system environment variables in angular. These env vars are hardcoded  in the environment file as suggested by Angular documentation. We could argue that it will be visible to our user in production anyway so it is not a bit deal if we expose it here.

Updated some functions that was originally not used.

### CODE REVIEW

File: webapp/src/client/app/shared/services/analytic.service.ts [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        .catch((error) => {
+          if (isDevMode()) {
+            console.error('Error sending event', error);
+          }
+        });
+    });
+  }
+}
+
+export interface ConnectionIndentifier {
```
Comment (pauldumais) [inline webapp/src/client/app/shared/services/analytic.service.ts:123]: Typo "Identifier" remove the 2nd letter "n".

Comment (irvingoujAtDevolution) [reply webapp/src/client/app/shared/services/analytic.service.ts]: updated!

**Review by kristahouse — APPROVED** [review]
LGTM

**Review by pauldumais — APPROVED** [review]

**Review by pauldumais — APPROVED** [review]
LGTM

**Comment by CBenoit** [comment]
note: The scope for this PR/commit should be `webapp` instead of `dgw` (should be adjusted when redacting the release notes, cc @thenextman)

**Pull request merged**